coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -130,3 +130,45 @@ def to_spacecase(string):
     return re.sub('(?<=[^\s])([A-Z])',
                   lambda match: ' ' + match.group(1),
                   string)
+
+
+def to_kebabcase(string):
+    """
+    Converts the given string to kebab-case.
+
+    >>> to_kebabcase('HelloWorld')
+    'hello-world'
```

Comment:
As @Siddhartha1234 I followed the haskell conventions. To follow the npm conventions for kebab-case, I think we can file another issue. Let me know how the changes look. @Siddhartha1234 @RohanVB